appservice/src/deploy/deployZip.ts

```diff
@@ -44,6 +44,16 @@ export async function deployZip(client: SiteClient, fsPath: string, configuratio
             const kuduClient: KuduClient = await getKuduClient(client);
             await kuduClient.pushDeployment.zipPushDeploy(fs.createReadStream(zipFilePath), { isAsync: true });
             await waitForDeploymentToComplete(client, kuduClient);
+            if (asp && asp.sku && asp.sku.tier && asp.sku.tier.toLowerCase() === 'basic') {
+                const deployments: number = (await kuduClient.deployment.getDeployResults()).length;
+                // if this is the first deployment, implement a 10 second delay for apps in a basic plan due to long start times
+                if (deployments === 1) {
```

Comment:
`if (deployments === 1) {` works reliably? I guess I figured there may be other "temp" deployments or something causing this number to be higher even for the first one

Reply:
I only tested it with a few apps, but new apps only had 1 deployment after it successfully completed.  

Since @fiveisprime seems to mostly be worried about tutorial followers, I don't think we're expecting it to fail so I think it'd be okay?